dask_image/ndinterp/__init__.py

```diff
@@ -381,3 +382,68 @@ def spline_filter1d(
     )
 
     return result
+
+
+def map_coordinates(image, coordinates, order=3,
+                    mode='constant', cval=0.0, prefilter=False):
+    """
+    Pre-map coordinates onto the chunks of 'image' and execute
+    `ndimage.map_coordinates` for every chunk.
+    """
+
+    coordinates = np.asarray(coordinates)
```

Comment:
This currently assumes that coordinates is a small enough array that it will fit in memory, but what if both ``image`` and ``coordinates`` are large arrays and would both not fit in memory? Or is it unlikely that this would be the case?